src/Integration/Concerns/TracesTrellisHttpRequests.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Vinelab\Tracing\Integration\Concerns;
+
+use Trellis\Clients\Facades\Intercept;
+use Vinelab\Tracing\Facades\Trace;
+use Vinelab\Tracing\Propagation\Formats;
+
+/**
+ * Enables tracing for Trellis inter-service communication over HTTP
+ */
+trait TracesTrellisHttpRequests
```

Comment:
Why would we include product specific traits in the package. How do we define Trellis in this context? Doesn't `TracesLucidArchitecture` suffice for integration?

Reply:
If we could keep this one out of the package, that would be ideal. 

As I was editing this yesterday, I kept thinking that this might become a maintenance problem considering I don't work on the product anymore. It's kinda sticking like a sore thumb here.

For example, if we were to provide test coverage for new traits, testing lucid events is possible as long as we use something like orchestral testbench but testing trellis http integration is not because the related package is private.

Reply:
Totally, I don't believe this fits here at all. And if `TracesLucidArchitecture` does not cater for what we are looking for, then what is it for?

Reply:
@Mulkave Seeing that there is another voice supporting this, can we remove Trellis stuff from here and settle on just the Lucid integration?

Reply:
Pushed this in a new commit to speed up the discussion. Feel free to revert it if needed, but I think we arrived at a good place now. BTW, one other notable change I made today is to support queuing lucid features here since a couple of services rely on this.

Reply:
so we still will be forced to copy paste some functions in each Trellis service where we want to have tracing.